SampleOfferWithoutReward.sol

```diff
@@ -90,8 +124,9 @@ contract SampleOfferWithoutReward {
             isContractValid = false;
     }
 
+    // TODO: This function is quite inconvenient. See withdraw() alternative.
     function getDailyPayment() {
-        if (msg.sender != contractor)
```

Comment:
Because this is a contract between the contractor and the client and nobody else.

Even if it's not a threat to the security of the contract, I would like to have checks for anything that is not supposed to happen.